tests/vm_test.go

```diff
@@ -169,7 +170,28 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		type vmiBuilder func() (*v1.VirtualMachineInstance, *cdiv1.DataVolume)
 
 		newVirtualMachineInstanceWithFileDisk := func() (*v1.VirtualMachineInstance, *cdiv1.DataVolume) {
-			return tests.NewRandomVirtualMachineInstanceWithFileDisk(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine), testsuite.GetTestNamespace(nil), corev1.ReadWriteOnce)
+			sc, foundSC := libstorage.GetRWOFileSystemStorageClass()
+			Expect(foundSC)
```

Comment:
```suggestion
			Expect(foundSC).To(Not(HaveOccurred()), "Filesystem storage is not present")
```
Sorry for my former (careless) suggestion. This time, again, I did not try this code out. Please see if this behaves properly with `foundSC=false`.

Reply:
@RoniKishner would you agree to add an explanatory string?

Reply:
I agree